src/layouts/LoginLayout/LoginLayout.test.tsx

```diff
@@ -39,21 +44,23 @@ beforeAll(() => {
 afterEach(() => {
   queryClient.clear()
   server.resetHandlers()
+  cleanup()
+  vi.clearAllMocks()
 })
 
 afterAll(() => {
   server.close()
-  jest.restoreAllMocks()
+  vi.restoreAllMocks()
 })
 
 describe('LoginLayout', () => {
   function setup() {
     server.use(
-      graphql.query('CurrentUser', (req, res, ctx) =>
-        res(ctx.status(200), ctx.data({ me: null }))
-      )
+      graphql.query('CurrentUser', (info) => {
+        return HttpResponse.json({ data: { me: null } }, { status: 200 })
```

Comment:
Curious on why this change?

Reply:
MSW v1 doesn't work with Vitest, and MSW v2 doesn't work with Jest. So as we're migrating we're also moving over to MSW v2

Reply:
Which has slightly different syntax